projects/libyuv/libyuv_fuzzer.cc

```diff
@@ -0,0 +1,80 @@
+/* # Copyright 2020 Google Inc. */
+/* # */
+/* # Licensed under the Apache License, Version 2.0 (the "License"); */
+/* # you may not use this file except in compliance with the License. */
+/* # You may obtain a copy of the License at */
+/* # */
+/* #      http://www.apache.org/licenses/LICENSE-2.0 */
+/* # */
+/* # Unless required by applicable law or agreed to in writing, software */
+/* # distributed under the License is distributed on an "AS IS" BASIS, */
+/* # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. */
+/* # See the License for the specific language governing permissions and */
+/* # limitations under the License. */
+/* # */
+/* ################################################################################ */
+
+#include <iostream>
+#include <vector>
+#include <cmath>
+
+#include <fuzzer/FuzzedDataProvider.h>
+#include "libyuv.h"
+#include "fuzz_common.h"
+
+using namespace std;
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+
+    FuzzedDataProvider fuzzed_data(data, size);
+
+    int range_size = sqrt((static_cast<int>(size) + 1) / 2);
```

Comment:
consider non-square?
if you come up with a randomized width that is less than size you can compute height